README.md

```diff
@@ -29,48 +29,37 @@ To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
-
-### Install
+To check this project you only need internet conection
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
+To get the best of this project, you can check it on different sizes, and test the navigation bar
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **lisandro seia**
 
-👤 **Author2**
+- GitHub: [@lisandro seia](https://github.com/lisandroseia)
+- LinkedIn: [@lisandro seia](https://www.linkedin.com/in/lisandro-seia-295120225/)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/lisandroseia/capstone1/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- I used  this [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),
+and I tried to adapt it to my topic.
+- I am extremely thankful with all my coding partners, who helped me a lot
+- I inspired in my dog, thats why i choosed this topic
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin wh is the author of the original design. Please try and fix that.
- [ ] Your live demo link is not working. Please try and fix that.